gdal/doc/source/tutorials/geotransforms_tut.rst

```diff
@@ -0,0 +1,37 @@
+.. _geotransforms_tut:
+
+================================================================================
+Geotransform Tutorial
+================================================================================
+
+Introduction to Geotransforms:
+-------------------------------
+A geotransform is an affine transformation from the image coordinate space (row, column),
+also known as (pixel, line)
+to the georeferenced coordinate space (projected or geographic coordinates).
+
+A geotransform consists in a set of 6 coefficients:
+----------------------------------------------------
+
+| ``GT(0)`` x-coordinate of the upper-left corner of the upper-left pixel.
+| ``GT(1)`` w-e pixel resolution / pixel width.
+| ``GT(2)`` row rotation (typically zero).
+| ``GT(3)`` y-coordinate of the upper-left corner of the upper-left pixel.
+| ``GT(4)`` column rotation (typically zero).
+| ``GT(5)`` n-s pixel resolution / pixel height (negative value for a north-up image).
+
+Transformation from image coordinate space to georeferenced coordinate space:
+-------------------------------------------------------------------------------
+
+| ``X_geo = GT(0) + X_pixel | GT(1) + Y_line | GT(2)``
```

Comment:
Haven't tested how it looks in .rst but the | character isn't appropriate for multiplication
```suggestion
| ``X_geo = GT(0) + X_pixel * GT(1) + Y_line * GT(2)``
```
Same for below formula

Reply:
Thanks, that's a result of an incautious search replace...